template/config/high_five.rb

```diff
@@ -5,6 +5,9 @@
   config.root = File.join(File.dirname(__FILE__), '..')
   config.destination = "www"
 
+  # This will run cordova prepare in this directory so you should set your deploy destination to cordova/www
+  # config.cordova_path "cordova"
```

Comment:
Can we also modify hi5 dist to use this as a hint for the platform_path if this is present and platform_path isn't? That would be super because then stuff like hi5 ios:set_version would all just work out of the box